helm-chart/binderhub/values.yaml

```
@@ -272,8 +272,8 @@ dind:
   initContainers: []
   daemonset:
     image:
-      name: docker
-      tag: 20.10.12-dind
+      repository: docker.io/library/docker
```

```suggestion
      image: docker.io/library/docker
```
`repository` is a better name for this property though, and matches other Helm charts

---

Wooops renaming the values property was a mistake